app/model/package.scala

```diff
@@ -63,6 +63,10 @@ trait UserMetadataLike {
   }
 }
 
+object UserMetadataLike {
+  val sortChronologically: ((String,String)) => String = (checkpoint) => checkpoint._2
```

Comment:
is sorting timestamps as strings good enough? I didn't research it much, just asking as I'd probably convert to an Instant before sorting

Reply:
Also I was a bit confused by this method. I assumed the function `sortChronologically` does sorting but it doesn't. It just discards the first value which is not a timestamp

Reply:
I think sorting strings is fine. They are all in the same format, so this seems the minimalist implementation.

Reply:
Ok, I can rename the function to be clearer of the intent - any suggestions?

Reply:
how about getting rid of the function and doing sth like:

```updatedNotification
  .checkpoints()
  .toSeq
  .map {
    case (keyName, timestamp) -> Instant.parse(timestamp)
  }.sort
```

Reply:
That will work. But the parsing implies that all the strings are in an expected format. And I thought one of the concerns was that sorting strings would fail if we didn't trust the format. (This might have been Tomasz's point though).

If the parsing does fail then we get an uncaught exception, whereas with the strings we just get timestamps in a log statement in the wrong order.

Reply:
I've realised too late it's only needed for logging, let's use strings then as you originally did and in a rare case it doesn't sort correctly it doesn't matter much.